acceptance/tests/custom_facts/having_multiple_facts_in_one_file.rb

```diff
@@ -0,0 +1,40 @@
+test_name 'C14893: Facter should handle multiple facts in a single file' do
+  tag 'risk:high'
+
+  fact_content = <<-EOM
+Facter.add(:test_fact1) do
+  setcode do
+    "test fact 1"
+  end
+end
+
+Facter.add(:test_fact2) do
+  setcode do
+    "test fact 2"
+  end
+end
+EOM
+
+  agents.each do |agent|
+    fact_dir = agent.tmpdir('facter')
+    fact_file = File.join(fact_dir, 'test_facts.rb')
+    create_remote_file(agent, fact_file, fact_content)
+    env = {'FACTERLIB' => fact_dir}
+
+    teardown do
+      on(agent, "rm -rf '#{fact_dir}'")
+    end
+
+    step "Agent: Verify test_fact1 from #{fact_file}" do
+      on(agent, facter('test_fact1', :environment => env)) do |facter_output|
```

Comment:
nit pick: The return from the Beaker `on` call is not an output object. It is an instance of [Beaker Result](http://www.rubydoc.info/github/puppetlabs/beaker/Beaker/Result) of which output streams are accessible from. So, the variable name `facter_output` is misleading. I am not advocating for changing it in this PR, just pointing it out.